rhel7/checks/oval/grub2_enable_fips_mode.xml

```diff
@@ -14,7 +14,6 @@
       <criteria operator="OR">
         <criterion test_ref="test_grub2_enable_fips_mode" comment="check for fips=1 in /etc/default/grub via GRUB_CMDLINE_LINUX" />
         <criteria operator="AND">
-          <criterion test_ref="test_grub2_default_exists" comment="check for GRUB_CMDLINE_LINUX_DEFAULT exists in /etc/default/grub" />
```

Comment:
@cipherboy why was this removed?

Reply:
Ah sorry. Meant to remove the or. It should only be an `AND`.

Reply:
@cipherboy I'm sorry, I don't understand. What should be removed? What shouldn't be removed? Why is this check removed? Are you sure you don't want to check existence of `GRUB_CMDLINE_LINUX_DEFAULT` in `/etc/default/grub`?